### ISSUE TITLE
Only expose URL explicitly

### ISSUE THREAD

**Issue by dpprdan** [ropensci/opencage#52]
The URL always contains the API key and we should prohibit leaking the key accidentally. Therefore we should only return the URL if the user requests it explicitly and only in an interactive session.

- [x] remove url from results output
- [x] make urlonly option only available in interactive mode (via #35)
- [x] document the risk of exposing URL

**Comment by maelle** [comment]
👌 

Do you want to assign this to yourself?

**Comment by dpprdan** [comment]
🤔 Should we obfuscate the key when printing URLs? Like

```r
oc_obfuscate_key <- function(key) {
  sub(
    pattern = "(key=)(\\w+)(\\w{5})",
    replacement = "\\1[your key ends with '\\3']",
    x = key
  )
}

oc_obfuscate_key("key=2e10e5e828262eb243ec0b54681d699a")
```

**Comment by freyfogle** [comment]
this sounds like an absolute nightmare for debugging.

**Comment by charliejhadley** [comment]
@dpprdan that seems like good practice, I made a PR for this

**Comment by dpprdan** [comment]
Also relevant for `oc_forward(return = "json_list", add_request = TRUE)`. Response contains API key.

**Comment by dpprdan** [comment]
Closed with #116
